src/adagrams.js

```diff
@@ -1,15 +1,95 @@
+import { letterPool, scoreChart } from 'constants';
+
+// helper function to create letter pool array
+export const createLetterPool = () => {
+  const letterPoolArray = [];
+
+  for (const letter in letterPool) {
+    for (let i = 0; i < letterPool[letter]; ++i) {
+      letterPoolArray.push(letter);
+    }
+  }
+
+  return letterPoolArray;
+};
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = createLetterPool();
+  const playerHand = [];
+  let counter = 0;
+
+  while (counter < 10) {
+    // access a letter via random index from letter pool
+    const randomIndex = Math.floor(Math.random() * (letterPool.length - 1));
```

Comment:
There's an off by one error here: since Math.random is between 0 inclusive and 1 exclusive, we don't need to decrease `letterPool.length` by one before calling `floor()`, otherwise we'll never access the last index.